nautobot_ssot/models.py

```diff
@@ -63,7 +180,8 @@ class Sync(BaseModel):
     dry_run = models.BooleanField(
         default=False, help_text="Report what data would be synced but do not make any changes"
     )
-    diff = models.JSONField(blank=True)
+    diff = models.JSONField(blank=True, null=True)
```

Comment:
I was wondering if we could replace the field, offering the same external interface, but with the compressed behavior internally.
if not, then I guess we should deprecate the old one at some point, so a warning message would be good